partner_report_open_invoices/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Open Invoices Reports",
+    "summary": "Open Invoices Reports",
+    "version": "8.0.1.0.0",
+    "category": "Accounting",
+    "description": """
```

Comment:
No, AFAIK.
If description field is not present, the description module is loaded from README
See https://github.com/odoo/odoo/commit/ec0b770ed3c4f4fbdf0f24930aaa05bcc5cb0b53 as reference.